index.html

```diff
@@ -144,9 +152,8 @@ <h3>Demos:</h3>
 </button>
 <code>macOSNotif({sounds:true})</code><br/>
 
-<script src="plugin/macOSNotif.js"></script>
 <script>
-    if(window.location.hostname != "macosnotif.js.org") window.location.replace("https://macosnotif.js.org");
+    // if(window.location.hostname != "macosnotif.js.org") window.location.replace("https://macosnotif.js.org");
```

Comment:
Is there a reason for this being commented out now?
_I presume this was done for testing?_